scipy/stats/_relative_risk.py

```diff
@@ -0,0 +1,197 @@
+
+from collections import namedtuple
+import numpy as np
+from scipy.special import ndtri
+
+
+ConfidenceInterval = namedtuple('ConfidenceInterval', ['low', 'high'])
+
+
+class RelativeRiskResult:
```

Comment:
does this need a docstring?

Reply:
Yup, done.

Reply:
Should it appear in the documentation? I can't find it now.

Reply:
At the bottom of the [main stats page](https://24801-1460385-gh.circle-artifacts.com/0/html-scipyorg/stats.html) is a link to [`scipy.stats.contingency.RelativeRiskResult`](https://24801-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.stats.contingency.RelativeRiskResult.html#scipy.stats.contingency.RelativeRiskResult)

Reply:
...but I pushed a change that move the location of the link up to just below the list of `contingency` functions.